### PR TITLE
Add sources to chart so the source of the chart is known

### PR DESCRIPTION
**Pull request by EugenMayer**
## Description

It is always hard to understand an chart without it's source, especially this chart since the values are hardly document (the entire space)
So it is useful to understand where the source code is, which is fairly hard to find right now (i had to ask in slack myself to be pointed here).

I would love to improve this for others to find it an easier way on https://artifacthub.io/packages/helm/projectcalico/tigera-operator and others. Thoughts?

## Related issues/PRs

https://calicousers.slack.com/archives/CPTH1KS00/p1642115657039500

## Todos

- [x] Tests
- [x] Documentation
- [x] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by EugenMayer** [comment]
I cannot currently sign the CLA since it failed during the first time i tried since i had a github rate api limit issue, know when clicking it i cannot see the agreement any longer. Can we fix this?

**Comment by lwr20** [comment]
now when clicking it i cannot see the agreement any longehttps://cla-assistant.io/projectcalico/calico?pullRequest=5451

**Comment by EugenMayer** [comment]
Now it worked (the link you gave me). Thank you - accepted

File: calico/_includes/charts/tigera-operator/Chart.yaml
Comment (lwr20) [inline calico/_includes/charts/tigera-operator/Chart.yaml:4]: Perhaps this might be a more useful link here?
```suggestion
home: https://projectcalico.docs.tigera.io/about/about-calico
```

File: calico/_includes/charts/tigera-operator/Chart.yaml
Comment (lwr20) [inline calico/_includes/charts/tigera-operator/Chart.yaml:5]: That's the Tigera icon, I think most people will be expecting the calico icon "Felix":
```suggestion
icon: https://projectcalico.docs.tigera.io/images/felix_icon.png
```

**Review by lwr20 — COMMENTED** [review]
Thank you so much for this.  A couple of suggestions made.

**Comment by EugenMayer** [comment]
Sure, you obviously know the URL way better then i do :) Happy to apply your suggestions. Done

**Review by lwr20 — APPROVED** [review]

**Review by fasaxc — APPROVED** [review]

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
